test/cypress/e2e/fixtures/texts.json

```diff
@@ -15,7 +15,7 @@
   "missingRights": "Du mangler rettigheter for å se denne tjenesten",
   "next": "Neste",
   "requiredField": "Feltet er påkrevd",
-  "requiredFieldLastName": "Du må fylle ut Nytt etternavn",
+  "requiredFieldLastName": "Du må fylle ut nytt etternavn",
   "requiredFieldDateFrom": "Du må fylle ut Dato for navneendring",
```

Comment:
Shouldnt Dato have lowercase `D` ? We should perhaps fix the shortname (as you mentioned in the description) as well? Or maybe I have misunderstood 😅

Reply:
Yeah, I agree - but that's defined in the `frontend-test` app, and I don't really see the need to fix that just to update this test (it would break other tests for a while until this is merged as well). Since the `shortName` was introduced because of this change to new error messages, I'm assuming app developers will make the smarter decision to use lowercase names most of the time (since it looks nicer).

Reply:
Ah, I didnt read the full file path here, and didnt realise this was just testdata. Not too worried about it then, fine to leave as is 🙂